README.md

```diff
@@ -11,6 +11,10 @@
   <i>CircleCI Orb</i>
 </h1>
 
+[![CircleCI](https://circleci.com/gh/secrethub/secrethub-circleci-orb.svg?style=shield)](https://circleci.com/gh/secrethub/secrethub-circleci-orb)
+[![Version]( https://img.shields.io/github/release/secrethub/secrethub-circleci-orb.svg)](https://github.com/secrethub/secrethub-circleci-orb/releases/latest)
```

Comment:
Should we add this one after the initial release?

Reply:
I think it already adds value, by displaying there is no version released yet.

That being said, I think we should strive to release very soon :rocket:

Reply:
Haha, okay sure.